frontend/ui/src/pages/Tags/TableRow.tsx

```diff
@@ -1,16 +1,20 @@
 import React, {useState, useCallback} from 'react';
 import _, {connect, ConnectedProps} from 'react-redux';
 
-import styles from './TableRow.module.scss';
 import {updateTag} from '../../actions/tags';
+
 import {Button, LinkButton} from 'components';
-import {ReactComponent as EditIcon} from 'assets/images/icons/edit.svg';
-import {ReactComponent as TrashIcon} from 'assets/images/icons/trash.svg';
 import ColorSelector from '../../components/ColorSelector';
 import Tag from '../../components/Tag';
 import {Tag as TagModel, TagColor} from 'model';
 import {Settings} from '../../reducers/data/settings';
 import {StateModel} from '../../reducers';
+
+import {ReactComponent as EditIcon} from 'assets/images/icons/edit-pencil.svg';
```

Comment:
I suppose the "right" name would be `EditPencilIcon` (I know it was already there but since we're touching it...)

Reply:
I will change it